utxo/src/storage/in_memory.rs

```diff
@@ -31,14 +30,16 @@ pub struct UtxosDBInMemoryImpl {
 }
 
 impl UtxosDBInMemoryImpl {
-    pub fn new(best_block: Id<GenBlock>, initial_utxos: BTreeMap<OutPoint, Utxo>) -> Self {
+    #[allow(dead_code)]
```

Comment:
When I see this I am always tempted to try to remove the attribute in order to check if it is really needed and why. 🙃 I think a comment explaining it would be nice.